packages/core/src/events/EventHandlers.ts

```diff
@@ -32,43 +27,40 @@ export class EventHandlers extends Handlers<
       select: {
         init: () => () => this.store.actions.setNodeEvent("selected", null),
         events: [
-          event({
-            name: "mousedown",
-            handler: rapidDebounce((_, id: NodeId) =>
+          defineEventListener(
```

Comment:
mm is there a reason why you changed this? I find the LHS version slightly more readable (though it's more verbose), but it's not a strong preference since you can look up the function and find what each parameter does). I would move the `defineEventListener` function up to the top of this file though since you're not using it anywhere else anyways.

Reply:
btw, we should convert the underlying data structure to an object rather than an array. it's much more readable and extensible.

Reply:
let's leave that change for another PR though

Reply:
I'm following a similar API as `addEventListener` which I think it's quite easily understood. I believe this should be kept in the utils package because the layers package makes use of Handlers as well

Reply:
ok, makes sense!